hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/request/bucket/OMBucketCreateRequest.java

```diff
@@ -98,8 +98,8 @@ public OMRequest preExecute(OzoneManager ozoneManager) throws IOException {
 
     BucketInfo.Builder newBucketInfo = bucketInfo.toBuilder();
 
-    // Set creation time.
-    newBucketInfo.setCreationTime(Time.now());
+    // Set creation time and modification time.
+    newBucketInfo.setCreationTime(Time.now()).setModificationTime(Time.now());
```

Comment:
Here do we need to get the value once, and set the same value for creatiionTime and ModificationTime. Because once after createBucket, there is a chance of these having different values.

long updateTime = Time.now(), and then use this in setting mofication and creation time.

Same as below.